README.md

```diff
@@ -1,6 +1,6 @@
 ## OMERO DSL Plugin
 
-The OMERO DSL plugin for Gradle provides a plugin named `dsl`.
+The OMERO DSL plugin for [Gradle](https://gradle.org/) provides a plugin named `dsl`.
```

Comment:
This is a nice touch that could be brought to other READMEs, e.g., Blitz plugin.

Reply:
Applying.